spec/v1/bundle.md

```diff
@@ -2,40 +2,63 @@
 
 Version: v1.0
 
-An attestation bundle is a collection of multiple attestations in a single
+An attestation Bundle is a collection of multiple attestations in a single
 file. This allows attestations from multiple different points in the software
 supply chain (e.g. Provenance, Code Review, Test Result, vuln scan, ...) to
 be grouped together, allowing users to make decisions based on all available
 information.
 
-**NOTE**: The bundle is not authenticated as a whole. Instead each individual
-attestation is authenticated using [DSSE]. As such, an attacker might be able
-to _delete_ an attestation without being detected. Predicates that follow
-[the monotonic principle] should not have any issues with this behavior.
+**NOTE**: The Bundle is not authenticated as a whole. Instead each individual
+attestation is authenticated using signature schemes like [DSSE]. As such,
```

Comment:
Nit: we should avoid using the term "signature schemes" here because that term is typically used to describe a set of key-gen/signing/verification algorithms.

In the rest of this PR we use signature method / signature format. Let's use those here, too? And ensure we're consistent throught.

Reply:
Good catch, thanks! Yes, I had been trying to avoid signature scheme for exactly the reasons you mention, but this one managed to sneak in. Will definitely change.

Reply:
I'm also a little iffy on saying "signature method like DSSE". Can we say "each individual attestation is authenticated using signatures recorded in an envelope like DSSE"?

Reply:
I was going to go with "signature format". That's a term that DSSE uses, too. But this discussion has got me thinking that we may want to clarify the distinction between scheme/method/format on the Envelope spec regardless.